samcli/commands/local/cli_common/invoke_context.py

```diff
@@ -387,7 +387,7 @@ def stderr(self) -> StreamWriter:
             Stream writer for stderr
         """
         stream = self._log_file_handle if self._log_file_handle else osutils.stderr()
-        return StreamWriter(stream, auto_flush=True)
+        return StreamWriter(stream, auto_flush=True, pretty_json=True)
```

Comment:
I think we should also offer an option for users to not prettify json, as expanding the JSON object does not always provide better UX.